src/GoogleTranslateConnector.ts

```diff
@@ -0,0 +1,51 @@
+import { Reshuffle, BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
+import TranslationServiceClient, { protos, v3 } from '@google-cloud/translate'
+
+export interface GoogleTranslateConnectorConfigOptions {
+  credentials: string
+}
+
+export default class GoogleTranslateConnector extends BaseConnector<
+  GoogleTranslateConnectorConfigOptions,
+  null
+  > {
+
+  private client: v3.TranslationServiceClient
+
+  constructor(app: Reshuffle, options?: GoogleTranslateConnectorConfigOptions, id?: string) {
+    super(app, options, id)
+    this.client = new v3.TranslationServiceClient()
+  }
+
+  async translate(
```

Comment:
1. This is quite complex to use, a lot of param to pass. We need to set defaults for most of those parameters.
As a developer, I care mostly about: text, source and target (mandatory) and other params optionals.
Note: you can set defaults like that: mimeType: string = "text/plain"

2. What is the projectId?Shouldn't be part of the constructor instead? same for location

3. text, source and target should probably be the first 3